crates/driver/src/infra/blockchain/gas.rs

```diff
@@ -42,9 +43,16 @@ impl GasPriceEstimator {
                     mempool.additional_tip_percentage,
                 )
             });
+        // Use the lowest max_fee_per_gas of all mempools as the max_fee_per_gas
+        let max_fee_per_gas = mempools
+            .iter()
+            .map(|mempool| eth::U256::from_f64_lossy(mempool.gas_price_cap))
```

Comment:
`gas_price_cap` should be probably updated to `U256`, not related to this PR

Reply:
#2301